vividus-tests/src/main/resources/story/integration/TableTransformers.story

```diff
@@ -183,3 +183,13 @@ Examples:
     \{transformer=FROM_EXCEL\, path=/data/complex-data.xlsx\, sheet=with separators\, range=B1:B2\, column=col2\}
 }
 {transformer=FILTERING, column.col1=.+}
+
+Scenario: Verify possibility to use the range of ALL cells
+Meta:
+    @issueId 919
+Then `<date1>` is equal to `line1 line4`
+Then `<date2>` is equal to `line2 line5`
+Then `<date3>` is equal to `line3 line6`
+Examples:
+{transformer=FROM_EXCEL, path=/data/range-all-cells.xlsx, sheet=data, range=B2:D4}
```

Comment:
do you really need one more excel file? can we use existing one? (just add a new sheet)

Reply:
remove it to complex-data.xlsx